src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPolyline.java

```diff
@@ -54,39 +54,39 @@ public FlaggedPolyline(final AtlasItem atlasItem)
     }
 
     @Override
-    public String getCountry()
+    public JsonObject asGeoJsonFeature(final String flagIdentifier)
     {
-        return this.country;
+        final JsonObject feature = this.atlasItem.asGeoJsonGeometry();
+        final JsonObject jsonProperties = this.atlasItem.getGeoJsonProperties();
+
+        jsonProperties.addProperty("flag:id", flagIdentifier);
+        jsonProperties.addProperty("flag:type", FlaggedPolyline.class.getSimpleName());
+        feature.add("properties", jsonProperties);
+        return feature;
     }
 
     @Override
-    public Iterable<Location> getGeometry()
+    public Rectangle bounds()
     {
-        return this.polyLine;
+        return this.atlasItem.bounds();
     }
 
     @Override
-    public Map<String, String> getProperties()
+    public String getCountry()
     {
-        return this.properties;
+        return this.country;
     }
 
     @Override
-    public JsonObject asGeoJsonFeature(final String flagIdentifier)
+    public Iterable<Location> getGeometry()
     {
-        final JsonObject feature = atlasItem.asGeoJsonGeometry();
-        final JsonObject properties = feature.getAsJsonObject("properties");
```

Comment:
This would return null and fail some tests after https://github.com/osmlab/atlas/pull/369